bin/scalingo_postfrontend

```diff
@@ -10,6 +10,14 @@ mkdir -p build/
 mv src/frontend/out build/frontend-out
 
 mv src/backend/* ./
-mv src/nginx/* ./
+
+# Replace ${MESSAGES_FRONTEND_PORT} to Scalingo mandatory PORT env
```

Comment:
Is $MESSAGES_FRONTEND_PORT well named? it's the URL of the whole app, not just the frontend. So $PORT would make sense ?